### PR TITLE
Added fixture for sample data

### PR DESCRIPTION
**Pull request by tasawernawaz**
ECOM-3482

* Added fixture file with sample data
* Added test for preventing fixture from getting stale.

@zubair-arbi @awais786 @ahsan-ul-haq

### CODE REVIEW

**Comment by ahsan-ul-haq** [comment]
:+1:

**Comment by tasawernawaz** [comment]
@rlucioni Please review this.

File: credentials/apps/credentials/fixtures/sample_data.json [hunk 1]
```diff
@@ -0,0 +1,136 @@
+[
+  {
+    "fields": {
+      "image": "signatories/1/test_sign.png",
```
Comment (rlucioni) [inline credentials/apps/credentials/fixtures/sample_data.json]: Does this image need to exist somewhere?

**Comment by rlucioni** [comment]
@tasawernawaz looks good, just one question about the path to the image file.

File: credentials/apps/credentials/fixtures/sample_data.json (hunk 1 above)
Comment (tasawernawaz) [inline credentials/apps/credentials/fixtures/sample_data.json]: It is used in rendering credential, although if this is not present still credential will be rendered.

**Comment by rlucioni** [comment]
:+1:

**Pull request merged**